src/markdown-pages/build-apps/customize-visualization-with-New-Relic-components.mdx

```diff
@@ -0,0 +1,408 @@
+---
+path: '/build-apps/custom-visualizations-and-the-new-relic-one-sdk'
+duration: '25 min'
+title: 'Custom visualizations and the New Relic One SDK'
+template: 'GuideTemplate'
+description: 'Customize your visualization'
+resources:
```

Comment:
Should we add the previous guide in this series as well?

Reply:
✅